### DISCUSSION TITLE
CREATE UNIQUE INDEX on sqlite tables all exactly same size regardless of table size

### DISCUSSION THREAD

**Discussion by mbelvadi2** [dbeaver/dbeaver#12155] [Q&A]
#### System information: 
- Windows 10 Pro
- DBeaver version 7.2.1
- Additional extensions -none

#### Describe the problem you're observing:
I created 8 tables that each have different amounts of data and different column counts/names.
I then created unique indexes for each.
I noticed as I was creating each index that dbeaver was reporting back exactly the same number of "Updated Rows" for every one of the 8 indexes, despite widely different row counts in their corresponding tables.
When I noticed it the first time, I thought it had to do with my running the 8 create index commands in parallel ("Execute statements in separate tabs") so I deleted all 8 indexes and re-created each one one at a time with a normal execute.
I got exactly the same result.
I have attached screenshots for the last two of the 8, with screenshot showing the report from the create index and then a count(*) of the corresponding table so you can see the counts for them.
![Screenshot 2020-10-02 215102](https://user-images.githubusercontent.com/28821980/94979671-81be4e80-04fa-11eb-940d-7e9dfcddadde.jpg)
![Screenshot 2020-10-02 215040](https://user-images.githubusercontent.com/28821980/94979672-81be4e80-04fa-11eb-9934-82b17d30c436.jpg)
![Screenshot 2020-10-02 215026](https://user-images.githubusercontent.com/28821980/94979674-81be4e80-04fa-11eb-8bb8-acddeef8b8c7.jpg)
![Screenshot 2020-10-02 214857](https://user-images.githubusercontent.com/28821980/94979675-8256e500-04fa-11eb-8d2f-c67c7e787573.jpg)

#### Include any warning/errors/backtraces from the logs
Nothing of relevance

**Comment by kseniiaguzeeva** [comment]
Thank you for the report

**Comment by flambchap** [comment]
Hello. SQLite documentation [states](https://www.sqlite.org/c3ref/changes.html) that they

>  count only changes made by INSERT, UPDATE or DELETE statement

and that 

> Executing any other type of SQL statement does not modify the value returned by this function.

This perfectly explains why you got the same result over and over again. I believe that this behavior is not a bug.
